contracts/MarketWithoutProtection.sol

```diff
@@ -1,16 +0,0 @@
-// SPDX-License-Identifier: MIT
```

Comment:
Having `MarketWithoutProtection` is useful for testing. Sure you want to remove it?

Reply:
Lets rename to mock or something?  We have another approach for this as per Don suggestion